iroha/Cargo.toml

```diff
@@ -25,6 +25,7 @@ maintenance = { status = "actively-developed" }
 [dependencies]
 iroha_derive = { version = "=0.1.0", path = "../iroha_macro/iroha_derive" }
 iroha_network = { version = "=0.1.0", path = "../iroha_network" }
+iroha_logger = { version = "=0.1.0", path = "../iroha_logger"}
```

Comment:
I think `iroha-log` will look better.

Reply:
I thought about it, but then `iroha_logger` introduces only the logger implementation. This is similar to `env_logger`, `simple_logger`, `pretty_env_logger` and etc. which are all based on `log` crate. So I guess it makes sense to also call it `iroha_logger`.